c/libr.c

```diff
@@ -86,6 +79,13 @@ static int ARCompIName( const void *key, const void *vbase );
 int (*CmpARRtn)( const void *, const void * ) = ARCompName;
 int (*CmpOMFRtn)( const void *, const void *, unsigned ) = memcmp;
 
+#if defined( _M_IX86 ) && defined(__WATCOMC__)
```

Comment:
What's the reason to move those lines from one place to other?

Reply:
It's just taking function definitions out of #ifdef:
ARCompName
ARCompIName
CmpARRtn
CmpOMFRtn
if _WATCOMC is not defined you'll get error without these functions